source/includes/usage-examples/write-code-examples.java

```diff
@@ -0,0 +1,77 @@
+# start-insert-one
+Document document = new Document("<field name>", "<value>");
+Mono<InsertOneResult> result = Mono.from(collection.insertOne(document));
+
+System.out.printf("Inserted 1 document with ID %s%n.", 
+      result.block().getInsertedId());
```

Comment:
Let's separate here and in other places the code returning a `Publisher` from the code that then uses Project Reactor to consume it:

```suggestion
Publisher<InsertOneResult> insertOneResultPublisher = collection.insertOne(document);
InsertOneResult result = Mono.from(insertOneResultPublisher).block();
System.out.printf("Inserted 1 document with ID %s.%n",
      result.getInsertedId());
```